### ISSUE TITLE
Can't start WebUI on Windows

### ISSUE THREAD

**Issue by seanvree** [Flexget/Flexget#1810]
- Flexget version: 2.10.41.dev
- Installation method: GIT
- OS and version:  Windows 10E
- Python ver:  2.7

### Expected behaviour:

Run WebUI

### Actual behaviour:

WebUI fails to run

### Steps to reproduce:
- Step 1: ...

#### Config:
```
tasks:
  some_task:
    rss: http://example.com
    template: tv


web_server:
  bind: 127.0.0.1 # IP V4
  port: 3539 # Valid port number
  ssl_certificate: 'c:\ssl\cert1.pem'
  ssl_private_key: 'c:\SSL\cert1.key'
  web_ui: yes # Web-UI can optionally be disabeled, only API will run
  base_url: Set a different base_url. Default is / (None)
```
  
#### Log:
```
2017-05-01 11:05 VERBOSE  check                         Pre-checked 13 configuration lines
2017-05-01 11:05 VERBOSE  check                         Checking config file `c:\Flexget\config.yml`
2017-05-01 11:05 VERBOSE  check                         Config passed check.
2017-05-01 11:05 VERBOSE  deluge                        Cannot use deluge from install directory because its python version doesn't match.
2017-05-01 11:06 ERROR    manager                       Cannot daemonize on windows
2017-05-01 11:06 INFO     scheduler                     No schedules defined in config. Defaulting to run all tasks on a 1 hour interval.
2017-05-01 11:06 INFO     scheduler                     Starting scheduler
2017-05-01 11:06 INFO     web_server_daemon                 Running web server at IP 127.0.0.1:3539
2017-05-01 11:06 INFO     web_server_daemon                 Initiating API
2017-05-01 11:06 INFO     web_server_daemon                 Registering WebUI
2017-05-01 11:06 CRITICAL webui                         Failed to start web ui, this can happen if you are running from GitHub version and forgot to run the web ui build, see http://flexget.com/wiki/Web-UI for instructions
2017-05-01 11:06 INFO     web_server                    Web interface available at https://127.0.0.1:3539/Set a different base_url. Default is / (None)
```

**Comment by cvium** [comment]
Your config is wrong... You haven't set `base_url`

**Comment by liiight** [comment]
I fixed the wiki, but note that blindly copying examples of the wiki is really not recommended

**Comment by seanvree** [comment]
I don't get it!?  What is wrong with my config file?

```
tasks:
  some_task:
    rss: http://mysite.com/myfeed.rss


web_server:
  bind: 127.0.0.1 # IP V4
  port: 3539 # Valid port number
  ssl_certificate: 'c:\ssl\cert1.pem'
  ssl_private_key: 'c:\SSL\cert1.key'
  web_ui: yes # Web-UI can optionally be disabeled, only API will run
  base_url: /flexget Set a different base_url. Default is / (None)
```

**Comment by seanvree** [comment]
@liiight Yeah, this is the one I'm using:

```
c:\Flexget>flexget daemon start
2017-05-01 12:01 INFO     scheduler                     No schedules defined in config. Defaulting to run all tasks on a 1 hour interval.
2017-05-01 12:01 INFO     scheduler                     Starting scheduler
2017-05-01 12:01 INFO     web_server_daemon                 Running web server at IP 127.0.0.1:3539
2017-05-01 12:01 INFO     web_server_daemon                 Initiating API
2017-05-01 12:01 INFO     web_server_daemon                 Registering WebUI
2017-05-01 12:01 CRITICAL webui                         Failed to start web ui, this can happen if you are running from GitHub version and forgot to run the web ui build, see http://flexget.com/wiki/Web-UI for instructions
2017-05-01 12:01 INFO     web_server                    Web interface available at https://127.0.0.1:3539/flexget
```

**Comment by liiight** [comment]
Bah, running from git...
Read the rest if the wiki on how to build ui
